File: Prog52a's/Prog52a.c
#include <stdio.h>
int main() {
    int len;
    int wid;
    
    printf("Type width: \n");     scanf("%d", &len);
    printf("Type length: \n");     scanf("%d", &wid);
    printf("Area is: %d \n", len*wid);
    printf("Perimeter is: %d \n", (len*2) + (wid*2));
    
    return 0;
}

File: Prog52a's/Prog52a.pl
print "Enter Length: "; $len = <STDIN>; chomp $len;
print "Enter Width: "; $wid = <STDIN>; chomp $wid;

$area = $len*$wid;
$perim = ($len*2) + ($wid*2);

print "Area: ", $area, "\n";
print "Perimeter: ", $perim, "\n";